### PR TITLE
Removed unused properties from carbon

### PR DESCRIPTION
**Pull request by nareshpr**

### CODE REVIEW

**Comment by nareshpr** [comment]
retest this please

File: core/src/main/java/org/carbondata/core/constants/CarbonCommonConstants.java [hunk 1]
```diff
@@ -126,25 +78,9 @@
    */
   public static final String STORE_LOCATION_DEFAULT_VAL = "../unibi-solutions/system/carbon/store";
```
Comment (ravipesala) [inline core/src/main/java/org/carbondata/core/constants/CarbonCommonConstants.java]: Change the path name `unibi-solutions`

File: conf/carbon.properties.template [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#################### Performance Configuration ##################
```
Comment (ravipesala) [inline conf/carbon.properties.template]: Pls remove molap.properties from `dev` folder.

**Comment by nareshpr** [comment]
Fixed review comments

File: processing/src/main/java/org/carbondata/processing/store/CarbonFactDataHandlerColumnar.java [hunk 3]
```diff
@@ -88,12 +88,6 @@
   private static final LogService LOGGER =
       LogServiceFactory.getLogService(CarbonFactDataHandlerColumnar.class.getName());
   /**
-   * decimalPointers
```
Comment (ravipesala) [inline processing/src/main/java/org/carbondata/processing/store/CarbonFactDataHandlerColumnar.java]: Why this property is removed? why are you always taking default value? Why not let user configure this value? Any reason?

**Comment by ravipesala** [comment]
LGTM

**Pull request merged**